Factory/ConnectionFactory.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace steevanb\SSH2Bundle\Factory;
+
+use steevanb\SSH2Bundle\Model\Profile;
+use steevanb\SSH2Bundle\Model\Connection;
+
+/**
+ * Service for SSH2 connections
+ */
+class ConnectionFactory
```

Comment:
Pattern Factory is a good one, but when there is just 2 lines of code, not sure it's usefull to create a service just for it.

Reply:
You did the same, I just move the Service\Connection.php class to Factory\ConnectionFactory.php